src/TodoList.jsx

```diff
@@ -0,0 +1,30 @@
+const todoList = [
+    {
+      id: 1,
+      title: 'Complete week 1 assignment'
+    },
+    {
+      id: 2,
+      title: 'Complete week 2 assignment'
+    },
+    {
+      id: 3,
+      title: 'Complete week 3 assignment'
+    }
+  ];
+
+const TodoList = () => {
+    return (
+        <>
+            <ul>
+            {todoList.map((todo) => {
+            return (
+                <li key={todo.id}>{`${todo.title}`}</li>
```

Comment:
```suggestion
                <li key={todo.id}>{todo.title}</li>
```
Actually it is unnecessary to use template literals here - there is a more straightforward way to achieve the same result. Even if you add` todo.id`, still there is no need for template literals. Look: <img width="1130" alt="Screenshot 2024-06-24 at 12 22 06" src="https://github.com/omk963/react-todo/assets/114008959/96a36b51-ac75-41c6-b25f-3f17fd49946e">